internal/slicer/slicer.go

```diff
@@ -72,33 +83,33 @@ func Run(options *RunOptions) (*Report, error) {
 		targetDirAbs = filepath.Join(dir, targetDir)
 	}
 
+	packages, err := fetchPackages(options)
```

Comment:
[Comment for reviewer]: Before, we were doing package fetching in two steps, checking if the package existed and then fetching it.

It makes sense for the refactor to consolidate both for the sake of readability but, apart from that reason, we also need this function for the chisel.db and for the FIPS PRs. The reason for the former is that we need a single place were we have all the data about packages and archives in order to include that information in the db. The reason for the latter is that we have to do more logic for archive selection and it is far simpler if that logic is all in one place.